.github/workflows/pypi-upload.yml

```diff
@@ -2,6 +2,7 @@ name: PyPI Upload
 
 on:
   push:
```

Comment:
I was under the impression that we were aiming to use `releases :`. Is that change coming soon? or this is just a leftover?

Reply:
A change is coming and I wanted to keep it the same as what we have in alpha but first I'm checking in a private repo to make sure the tags don't get mixed up, meaning releasing a tag for alpha shouldn't release the main machine stats.